js/script.js

```diff
@@ -24,3 +26,23 @@ const addToScore = () => {
 const updateScoreBoard = () => {
     scoreBoard.textContent = score.toString();
 }
+
+/**
+ * Random zombie selector
+ */
+function pickRandom() {
+    let randomArray = [
+
```

Comment:
Looks good - but no need for the space here - sorry to be picky

Reply:
Good spot. That's done.